lib/sis_scraper.rb

```diff
@@ -3,18 +3,16 @@
 class SISScraper
   ALPHA = ('A'..'Z').to_a
   SIS = 'https://sisguest.case.edu'
-  SEM_LIST_POS = 1 		# position of the select list that determines the semester
-  LEN_DEPT_NAME = 4		# Standard length of a department abbreviation
- 
+  SEM_LIST_POS = 1 # position of the select list that determines the semester
+  LEN_DEPT_NAME = 4	# Standard length of a department abbreviation
+
   def initialize(download_dir)
-    @download_dir=download_dir
+    @download_dir = download_dir
     @browser = Watir::Browser.new :firefox
   end
 
-  def download_all_sis_data(user,pword)
+  def download_all_sis_data
```

Comment:
Assignment Branch Condition size for download_all_sis_data is too high. [15.17/15]